Commons/src/main/java/eu/neclab/ngsildbroker/commons/tools/HttpUtils.java

```diff
@@ -254,20 +253,29 @@ public static Uni<RestResponse<Object>> generateReply(HttpServerRequest request,
 
 	}
 
+/// modifiying coding 
 	private static Uni<String> getReplyBody(List<String> acceptHeader, int endPoint,
 			ArrayListMultimap<String, String> additionalHeaders, Object expanded, boolean forceArrayResult,
-			Context ldContext, List<Object> contextLinks, String geometryProperty) {
+			Context ldContext, List<Object> contextLinks, String geometryProperty, String option) {
 		String replyBody;
+		String s = null;
 		int sendingContentType = parseAcceptHeader(acceptHeader);
 		Map<String, Object> compacted;
 		try {
 			compacted = JsonLdProcessor.compact(expanded, contextLinks, ldContext, opts, endPoint);
+			if (option != null && option.equals(NGSIConstants.QUERY_PARAMETER_CONCISE_VALUE)) {
+				conciseRepresentation(compacted, null, "");
```

Comment:
Why are you using empty string, you can use null.

Reply:
because I am checking empty string in further execution